peachjam/templates/peachjam/_document_table_form.html

```diff
@@ -0,0 +1,113 @@
+{% load i18n %}
+<form method="get"
+      id="filter-form"
+      hx-include="#filter-form"
+      hx-indicator=".htmx-progress"
+      hx-target="#filter-form"
+      hx-swap="outerHTML">
+  <div class="row">
+    <div class="col-lg-3 d-none d-lg-block">
+      <div>
+        {% block desktop-taxonomy-list %}{% endblock %}
+      </div>
+      <div class="pocketlaw-hidden">
+        <ul class="list-group">
+          <li class="position-relative list-group-item bg-light d-flex justify-content-between align-items-center">
+            <strong>{% trans 'Filters' %}</strong>
+          </li>
+          {% for facet_name, facet in facet_data.items %}
+            {% if facet.options %}
+              <li class="list-group-item">
+                <div class="d-flex justify-content-between mb-2">
+                  <strong>{{ facet.label }}</strong>
+                  <div class="d-flex align-items-center">
+                    <a href="{{ request.path }}"
+                       hx-get="{{ request.path }}"
+                       hx-params="not {{ facet_name }}"
```

Comment:
@longhotsummer after much fiddling, I found outh that apparently htmx does its own thing with url params, this is the correct way to handle the clearing,